src/lib/index.ts

```diff
@@ -2,6 +2,17 @@ import fetch from 'node-fetch';
 import path from 'path';
 import execSh from 'exec-sh';
 import { specmaticJarPathLocal } from '../config';
+import fs from 'fs';
+
+
+
+export const setSpecmaticEnvVariable = (name: string, value: string, fileName: string) => {
+  const file = require("../../" + fileName)
+  file.environments.local.variables[name] = value
+  fs.writeFileSync("specmatic.json", JSON.stringify(file, null, 2))
+}
+
+export const setEnvVariable = setSpecmaticEnvVariable;
```

Comment:
I would say lets skip this alias. There are aliases created when an older name is already in use. So that if some consumer project is already using that method, their changes don't break when we change the name of the method. But in our case, because its a new method, no one is using it yet, so we can skip it. and stick with `setSpecmaticEnvironement`

Reply:
Added the required change